.github/workflows/integration-tests.yml.in

```diff
@@ -387,6 +387,13 @@ jobs:
           # Run runtime tests serially to avoid race condition with cache handling
           python3 -m pytest runtime
 
+      - name: Run Proton tests
+        env:
+          LD_LIBRARY_PATH: "/opt/rocm/lib:$LD_LIBRARY_PATH"
```

Comment:
We don't need to set this `LD_LIBRARY_PATH`. You can just drop the `env:` part here.